recipes/pyzmq/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+ZMQ="${PREFIX}"
+
+if [ "$(uname)" == "Darwin" ]; then
+    ZMQ=bundled
+    export MACOSX_VERSION_MIN=10.7
+    export CC="clang"
+    export MACOSX_DEPLOYMENT_TARGET=10.6
+    export DYLD_LIBRARY_PATH="$PREFIX/lib:$DYLD_LIBRARY_PATH"
```

Comment:
moving this line to the end should prevent it from interfering with the `xcodebuild` command